examples/braket_features/Getting_Devices_and_Checking_Device_Properties.ipynb

```diff
@@ -78,7 +78,7 @@
   },
```

Comment:
<p>This doesn't include Garnet, is that a follow up? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/578/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Yes